_example/simple-echo/main.go

```diff
@@ -19,6 +21,7 @@ func completer(in prompt.Document) []prompt.Suggest {
 func main() {
 	in := prompt.Input(">>> ", completer,
 		prompt.OptionTitle("sql-prompt"),
+		prompt.OptionInitialText(strings.Join(os.Args[1:], " ")),
```

Comment:
Could you revert the changes of this example? I want to keep this example simple.